components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCache.java

```diff
@@ -178,26 +232,48 @@ private String replaceFromTokenId(String keyValue) {
      * @param id to clear cache.
      */
     private void clearFromSessionStore(String id) {
+
         SessionDataStore.getInstance().clearSessionData(id, AUTHORIZATION_GRANT_CACHE_NAME);
     }
 
     /**
      * Retrieve cache entry from SessionDataStore
+     *
      * @param id session data key
      * @return
      */
     private AuthorizationGrantCacheEntry getFromSessionStore(String id) {
+
         return (AuthorizationGrantCacheEntry) SessionDataStore.getInstance().getSessionData(id,
                 AUTHORIZATION_GRANT_CACHE_NAME);
     }
 
     /**
      * Store cache entry in SessionDataStore
-     * @param id session data key
+     *
+     * @param id    session data key
      * @param entry cache entry to store
      */
     private void storeToSessionStore(String id, AuthorizationGrantCacheEntry entry) {
+
         SessionDataStore.getInstance().storeSessionData(id, AUTHORIZATION_GRANT_CACHE_NAME, entry);
     }
 
+    private void logClaims(String prefix, Map<ClaimMapping, String> userAttributes) {
```

Comment:
@senthalan shall we include log.isDebugEnabled() check in this logic ? That way we can easily alter the behavious of this function in future

Reply:
For this to be done, add a String parameter to this method. That will say where we extracted claims.